tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3343,28 +3340,25 @@ def runTest(self):
                 assert diff > 0, "Failed to cause TX drop on port {}".format(self.dst_port_id)
             xmit_counters_base = xmit_counters
             # Find a separate flow that uses alternate queue
-            max_iters = 50
-            for i in range(max_iters):
+            for index, (second_pkt, _) in enumerate(pkt_list):
```

Comment:
syntax error, after apply this line to latest 202205:
```
======================================================================
ERROR: sai_qos_tests.LossyQueueVoqTest
----------------------------------------------------------------------
Traceback (most recent call last):
  File \"saitests/py3/sai_qos_tests.py\", line 3378, in runTest
    for index, (second_pkt, _) in enumerate(pkt_list):
ValueError: too many values to unpack (expected 2)
```

Reply:
Hi ,
The recent merge of PR9362 in 202205 branch caused this error , I have updated this PR to fix that issue .